Document XSLTProcessor.* (#25563)
mdn/content: files/en-us/web/api/xsltprocessor/transformtodocument/index.md

---
title: XSLTProcessor.transformToDocument()
slug: Web/API/XSLTProcessor/transformToDocument
page-type: web-api-instance-method
browser-compat: api.XSLTProcessor.transformToDocument 
---

{{APIRef("XSLT")}}

> The `transformToDocument()` method of the {{domxref("XSLTProcessor")}} interface transforms the {{DOMxRef("Node")}} source given in parameter using the XSLT stylesheet associated with `XSLTProcessor` with the {{domxref("XSLTProcessor.importStylesheet")}}.

Reviewer: ```suggestion
The `transformToDocument()` method of the {{domxref("XSLTProcessor")}} interface transforms the provided {{DOMxRef("Node")}} source to a {{domxref("Document")}} using the XSLT stylesheet associated with `XSLTProcessor`.
```
